src/components/KeyboardShortcutsModal.js

```diff
@@ -32,57 +32,22 @@ const defaultProps = {
     isShortcutsModalOpen: false,
 };
 
-class KeyboardShortcutsModal extends React.Component {
-    componentDidMount() {
-        this.subscribedOpenModalShortcuts = [];
-
-        const openShortcutModalConfig = CONST.KEYBOARD_SHORTCUTS.SHORTCUT_MODAL;
-        this.unsubscribeShortcutModal = KeyboardShortcut.subscribe(
-            openShortcutModalConfig.shortcutKey,
-            () => {
-                if (this.props.isShortcutsModalOpen) {
-                    return;
-                }
-
-                ModalActions.close();
-                KeyboardShortcutsActions.showKeyboardShortcutModal();
-            },
-            openShortcutModalConfig.descriptionKey,
-            openShortcutModalConfig.modifiers,
-            true,
-        );
-
-        if (this.props.isShortcutsModalOpen) {
-            // The modal started open, which can happen if you reload the page when the modal is open.
-            this.subscribeOpenModalShortcuts();
-        }
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!prevProps.isShortcutsModalOpen && this.props.isShortcutsModalOpen) {
-            this.subscribeOpenModalShortcuts();
-        } else if (prevProps.isShortcutsModalOpen && !this.props.isShortcutsModalOpen) {
-            // Modal is closing, remove keyboard shortcuts
-            this.unsubscribeOpenModalShortcuts();
-        }
-    }
-
-    componentWillUnmount() {
-        if (this.unsubscribeShortcutModal) {
-            this.unsubscribeShortcutModal();
-        }
-        this.unsubscribeOpenModalShortcuts();
-    }
+function KeyboardShortcutsModal(props) {
```

Comment:
also based on the latest guidance [we should Destructuring props  ](https://github.com/Expensify/App/blob/c24579548097a7cb0dfc2093052086f4c01251ed/contributingGuides/STYLE.md)